docs/installation/community-operator.md

```diff
@@ -156,9 +198,8 @@ Refer to the [Grafana Deployment Overview](https://github.com/sustainable-comput
 
 ## Frequently Asked Questions
 
-# Will Kepler work on earlier releases of OpenShift
+### Will Kepler work on earlier releases of OpenShift?
```

Comment:
Could we also add a section on 

* how do I set nodeSelectors and tolerations ?
* Where is kepler pod deployed?